build.sbt

```
@@ -43,21 +41,15 @@ reporterConfig := reporterConfig.value.withColumnNumbers(true)
 
 reporterConfig := reporterConfig.value.withShowLegend(true)
 
-//wartremoverErrors ++= Warts.unsafe
-
-wartremoverWarnings ++= Warts.all
-
-enablePlugins(JettyPlugin)
-
-containerArgs := Seq(
-  "--config",
-  "jetty.xml"
-)
-
-containerPort := 8444
-
 compileOrder := CompileOrder.ScalaThenJava
 
+val otelAgentJar = Def.task {
+  (Runtime / managedClasspath).value
+    .find(_.data.getName.contains("opentelemetry-javaagent"))
+    .map(_.data.getAbsolutePath)
+    .getOrElse("")
```

If we fail to find an agent, perhaps we should log a warning?
```suggestion
    .getOrElse { 
           log.error("OpenTelemetry Java agent JAR not found on Runtime managedClasspath.")
    }
```